praw/models/reddit/submission.py

```diff
@@ -341,6 +341,39 @@ def unspoiler(self):
         """
         self.thing._reddit.post(API_PATH["unspoiler"], data={"id": self.thing.fullname})
 
+    def update_crowd_control_level(self, level: Union[str, int]):
```

Comment:
I annotated it that way because it reflects that fact that it doesn't matter if you the data is  `{"level": 1}` or `{"level": "1"}`, because the Requests [formats](https://github.com/praw-dev/praw/pull/1753/files#diff-9b2db9631b5b5e5a8571dc9f43632b1d2196f79dc2a274b23e9bc926b51f5b98R109) the two in the same way. On the other hand, users that type check their scripts before running them might want it to just be an int, so I could change it if you'd like.

Reply:
Please, just because that is what reddit has documented for this endpoint.